CHANGELOG.adoc

```diff
@@ -24,11 +24,14 @@
 
 === Breaking changes
 
+- https://github.com/eclipse-sirius/sirius-web/issues/4341[#4341] [diagram] Display diagram contextual menu (a.k.a the Palette) with right click instead of left click.
+It is not an API breaking change but a breaking change from the end-user point of view.
+
 
 === Dependency update
 
 - [releng] Upgrade `turbo` to version 2.3.3 (from 1.13.3 before).
-The `start` task used in our turbo configuration is now marked as `persistent` and as a result we have separated the `start` script from `sirius-web` used to run the frontend in dev from from the `start` script from other packages used to build them in development mode by remaning the later to `build-dev`.
+The `start` task used in our turbo configuration is now marked as `persistent` and as a result we have separated the `start` script from `sirius-web` used to run the frontend in dev from from the `start` script from other packages used to build them in development mode by remaining the later to `build-dev`.
```

Comment:
Isn't this word supposed to be `renaming`?

Reply:
good catch ;)